operations/patch_module.go

```diff
@@ -84,13 +84,18 @@ func PatchSetModule() cli.Command {
 			moduleBranch, err := getModuleBranch(module, proj)
 			if err != nil {
 				grip.Error(err)
-				mods, merr := ac.GetPatchModules(patchID, existingPatch.Project)
+				mods, projectIdentifier, merr := ac.GetPatchModules(patchID, existingPatch.Project)
 				if merr != nil {
 					return errors.Wrap(merr, "errors fetching list of available modules")
 				}
-
+				if len(mods) == 0 {
+					return errors.Errorf("Project '%s' has no configured modules. Specify different project or "+
+						"see the evergreen configuration file for module configuration.",
+						projectIdentifier)
+				}
 				if len(mods) != 0 {
-					grip.Noticef("known modules includes:\n\t%s", strings.Join(mods, "\n\t"))
+					return errors.Errorf("Could not find module named '%s' for project '%s'; specify different project or select correct module from:\n\t%s",
+						module, projectIdentifier, strings.Join(mods, "\n\t"))
 				}
 
 				return errors.Errorf("could not set specified module: \"%s\"", module)
```

Comment:
It seems like this line would never be reached since it will always hit either `if len(mods) == 0` or  `if len(mods) != 0` and will always return an error.

Reply:
yep, fixed